.github/workflows/testing.yml

```diff
@@ -46,6 +47,9 @@ jobs:
 
       - name: Setup
         uses: ./.github/actions/setup
+        with:
+          pinecone_api_key: ${{ secrets.PINECONE_API_KEY }}
```

Comment:
How were we calling `Setup` without passing the API key previously? I thought it was required. 🤔

Reply:
Hm, looking at the `Setup` action itself (`setup/action.yml`), I don't think we actually need the API key here, because `Setup` really only installs node version and linting and whatnot; it doesn't interact with Pinecone.

Let me try to remove it here...

Reply:
removed :)